data-loading/src/test/java/no/sikt/nva/data/report/api/etl/SingleObjectDataLoaderTest.java

```diff
@@ -245,28 +258,29 @@ private URI setupExistingResourceInGraph(IndexDocumentWithConsumptionAttributes
      * the graph to a database and potentially return multiple graphs if a query is too broad. For example, where the
      * SPARQL query has GRAPH ?g rather than a specific named graph.
      */
-    private URI registerGraphForPostTestDeletion(UnixPath objectKey) {
-        graph = GraphName.newBuilder()
-                    .withBase("example.org")
-                    .fromUnixPath(objectKey)
-                    .build()
-                    .toUri();
+    private URI registerGraphForPostTestDeletion(URI uri) {
+        graph = URI.create(uri + ".nt");
         return graph;
     }
 
-    private UnixPath setupExistingObjectInS3(String folder) throws IOException {
-        var candidateDocument = IndexDocumentWithConsumptionAttributes.from(randomIndexDocument());
-        var objectKey = UnixPath.of(folder, constructFileIdentifier(candidateDocument
+    private UnixPath setupExistingObjectInS3(String folder,
+                                             IndexDocumentWithConsumptionAttributes document)
+        throws IOException {
+        var objectKey = UnixPath.of(folder, constructFileIdentifier(document
                                                                         .consumptionAttributes()
                                                                         .documentIdentifier()));
-        s3Driver.insertFile(objectKey, candidateDocument.toJsonString());
+        s3Driver.insertFile(objectKey, document.toJsonString());
         return objectKey;
     }
 
     private IndexDocument randomIndexDocument() {
         var identifier = UUID.randomUUID();
         return IndexDocument.builder()
-                   .withId(UriWrapper.fromHost(HOST).addChild(identifier.toString()).getUri())
+                   .withId(UriWrapper
+                               .fromHost(HOST)
```

Comment:
`NVI_CANDIDATES_FOLDER` is a bit misleading to read here, even though it doesn't matter. Rename constant to `NVI_PATH` and give some other value?